srcpkgs/heimdal/template

```diff
@@ -0,0 +1,32 @@
+# Template file for 'heimdal'
+pkgname=heimdal
```

Comment:
This package has content headers of various other packages and binaries as well, it is not suitable for inclusion